certificate-print-service/app/src/main/java/se/inera/intyg/certificateprintservice/application/print/GeneratePrintService.java

```diff
@@ -0,0 +1,34 @@
+package se.inera.intyg.certificateprintservice.application.print;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import se.inera.intyg.certificateprintservice.application.print.converter.PrintCertificateRequestConverter;
+import se.inera.intyg.certificateprintservice.application.print.dto.PrintCertificateRequestDTO;
+import se.inera.intyg.certificateprintservice.application.print.dto.PrintCertificateResponseDTO;
+import se.inera.intyg.certificateprintservice.pdfgenerator.PrintCertificateGenerator;
+
+@Service
+@RequiredArgsConstructor
+public class GeneratePrintService {
+
+  private final PrintCertificateGenerator printCertificateGenerator;
+  private final PrintCertificateRequestConverter printCertificateRequestConverter;
+
+  public PrintCertificateResponseDTO get(PrintCertificateRequestDTO request) {
+    validateRequest(request);
+    final var printCertificateRequest = printCertificateRequestConverter.convert(request);
```

Comment:
Variable is named printCertificateRequest but it's a certificate that's returned. Naming could be updated for variable and make more clear what converter converts to